packages/app/src/specs/SpecListGitInfo.vue

```diff
@@ -6,22 +6,23 @@
     <Tooltip
       placement="top"
       class="h-full grid items-center"
-      :disabled="!classes.showTooltip"
     >
       <component
         :is="classes.icon"
         :class="classes.iconClasses"
       />
       <template
-        v-if="classes.showTooltip"
         #popper
       >
         <div>
           <p class="max-w-sm text-sm truncate overflow-hidden">
```

Comment:
Can we conditionally render this if there is a tooltip message to show?

Reply:
Since the default was `null` I assumed there was cases when there wasn't content.

Is there always a tooltip even when the SCM is not available and/or git is not used in the project?

Reply:
If there's no SCM data available, this whole component won't be rendered
https://github.com/cypress-io/cypress/blob/06a02831e71432bd3837b9f4824c109c77ccbe52/packages/app/src/specs/SpecsList.vue#L105